fastlane/lib/fastlane/actions/pilot.rb

```diff
@@ -9,6 +9,7 @@ def self.run(values)
         values[:changelog] ||= changelog if changelog
 
         values[:ipa] ||= Actions.lane_context[SharedValues::IPA_OUTPUT_PATH]
+        values[:ipa] = File.expand_path(values[:ipa]) if values[:ipa]
```

Comment:
Ohhh good point, I think your original change is fine then 👍  I just didn't realize that values[:ipa] could already exist at that point.